main.go

```diff
@@ -33,9 +33,11 @@ func init() {
 func main() {
 	var namespace string
 	var metricsAddr string
+	var syncPeriod int
 	var enableLeaderElection bool
 	flag.StringVar(&namespace, "namespace", "", "Namespace within which CRD controller is running.")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.IntVar(&syncPeriod, "sync-period", int(2*time.Minute), "Sync every sync-period seconds.")
```

Comment:
int(2*time.Minute) is sync-period **nanoseconds**, not seconds.
We need to convert nanoseconds to seconds or update the flag description.

Reply:
Thanks. Fixed it.